src/controllers/productsController.js

```diff
@@ -0,0 +1,19 @@
+const { asyncWrap } = require('../utils/errorHandler');
+const productsService = require('../services/productsService');
+
+const getProductInfo = asyncWrap(async (req, res) => {
+  const { productId } = req.params;
+
+  const productInfo = await productsService.getProductInfo(productId);
+
+  if (productInfo.length === 0) {
+    return res.status(200).json({
+      data: productInfo,
+      message: 'The requested information is not found',
+    });
+  }
```

Comment:
상세의 경우 url로 특정 자원을 가리키기 때문에 해당 자원이 없을 경우 404 코드가 더 적절해 보입니다.
```suggestion
  if (productInfo.length === 0) {
    return res.status(404).json({
      data: productInfo,
      message: 'The requested information is not found',
    });
  }
```

Reply:
해당처리의 경우 Dao에서 해둔 상황이어서, 코드 제외하도록 하겠습니다!